x/msgfees/client/cli/tx.go

```diff
@@ -134,3 +137,42 @@ $ %[1]s tx msgfees remove "removing" "removing MsgWriterRecordRequest fee" 10nha
 	cmd.Flags().String(FlagMinFee, "", "additional fee for msg based fee")
 	return cmd
 }
+
+func GetUpdateUsdConversionRateProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "usd-conversion-proposal [title] [description] [rate] [deposit]",
+		Aliases: []string{"ucp", "u-c-p"},
+		Args:    cobra.ExactArgs(4),
+		Short:   "Submit a update usd conversion rate proposal along with an initial deposit",
+		Long: strings.TrimSpace(`Submit a update usd conversion rate proposal along with an initial deposit.
+The conversion rate is the cost for 1 unit of hash (1,000,000,000nhash).  Example: $1.023 would be represented as 1023 and $0.023 as 23.
+`),
+		Example: fmt.Sprintf(`$ %[1]s tx msgfees usd-conversion-proposal "updating usd conversion" "updating usd conversion to $1.234"  1234 1000000000nhash
```

Comment:
I would change the description slightly so it doesn't look exactly like the title. I thought this was a typo in the command at first because of how similar they are. Maybe something along the lines of "changes the usd conversion to $1.234."